### PR TITLE
HSEARCH-4290 Add an example to the documentation using ".bridgedElement().name()" in a property binder

### PR DESCRIPTION
**Pull request by yrodiere**
https://hibernate.atlassian.net/browse/HSEARCH-4290

### CODE REVIEW

File: documentation/src/main/asciidoc/reference/mapper-orm-bridge-propertybridge.asciidoc [hunk 1]
```diff
@@ -263,13 +263,46 @@ include::{sourcedir}/org/hibernate/search/documentation/mapper/orm/bridge/proper
...
+getting the property name and using it as a field name.
+
+.Naming a field after the property being bound in a `PropertyBinder`
+====
+[source, JAVA, indent=0, subs="+callouts"]
+----
+include::{sourcedir}/org/hibernate/search/documentation/mapper/orm/bridge/propertybridge/bridgedelement/InvoiceLineItemsSummaryBinder.java[tags=include]
+----
+<1> Use the binding context to get the bridged element.
+<2> Use the name of the property as the name of a declared field.
```
Comment (fax4ever) [inline documentation/src/main/asciidoc/reference/mapper-orm-bridge-propertybridge.asciidoc:288]: It is almost clear, but I would write something more explicit, like:
> Use the name of the property to get the access to the `objectField`

**Review by fax4ever — APPROVED** [review]
LGTM,
just an optional comment

File: documentation/src/main/asciidoc/reference/mapper-orm-bridge-propertybridge.asciidoc (hunk 1 above)
Comment (yrodiere) [reply documentation/src/main/asciidoc/reference/mapper-orm-bridge-propertybridge.asciidoc:288]: That's not right, though. We're not really "getting access" to the field, we're defining an index field. It doesn't exist before this line.

I'll clarify that I mean "index" field.

**Comment by yrodiere** [comment]
Merged, thanks

**Pull request merged**